src/panels/lovelace/editor/select-view/hui-dialog-select-view.ts

```diff
@@ -0,0 +1,79 @@
+import {
+  html,
+  LitElement,
+  TemplateResult,
+  customElement,
+  property,
+  css,
+  CSSResult,
+} from "lit-element";
+import "@polymer/paper-item/paper-item";
+import "../../../../components/dialog/ha-paper-dialog";
+// tslint:disable-next-line:no-duplicate-imports
+import { HaPaperDialog } from "../../../../components/dialog/ha-paper-dialog";
+
+import { SelectViewDialogParams } from "./show-select-view-dialog";
+import { PolymerChangedEvent } from "../../../../polymer-types";
+
+@customElement("hui-dialog-select-view")
+export class HuiDialogSelectView extends LitElement {
+  @property() private _params?: SelectViewDialogParams;
+
+  public async showDialog(params: SelectViewDialogParams): Promise<void> {
+    this._params = params;
+    await this.updateComplete;
+  }
+
+  protected render(): TemplateResult | void {
+    if (!this._params) {
+      return html``;
+    }
+    return html`
+      <ha-paper-dialog
+        with-backdrop
+        opened
+        @opened-changed="${this._openedChanged}"
+      >
+        <h2>Choose a view</h2>
+        ${this._params!.lovelace!.config.views.map((view, index) => {
```

Comment:
Can we copy the code from Cast that reserved space and shows icons if any of the views have an icon defined?

Reply:
Also let's also wrap it in a listbox so it's keyboard accessible. (Again see cast)